read-fonts/src/traversal.rs

```diff
@@ -149,6 +157,7 @@ impl<'a> FieldType<'a> {
     ///
     /// The `resolver` argument is a function that takes an offset and resolves
     /// it.
+    //TODO: rename this to array_of_offsets?
```

Comment:
+1 to the todo

Reply:
agree, going to do this in a separate PR to avoid cluttering this one up with a bunch of renames.